build.sbt

```diff
@@ -23,7 +23,7 @@ lazy val commonSettings = Seq(
   organization := "com.cognite",
   organizationName := "Cognite",
   organizationHomepage := Some(url("https://cognite.com")),
-  version := "2.1.4-SNAPSHOT",
+  version := "2.2.0",
```

Comment:
This PR changes an existing behaviour in spark datasource. `map() as metadata` was ignored. now, it will write empty map.